host/xtest/regression_4000.c

```diff
@@ -5543,4 +5543,335 @@ static void xtest_tee_test_4013(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 4013, xtest_tee_test_4013,
 		"Test generation of device unique TA keys");
+
+static void xtest_tee_test_4014(ADBG_Case_t *c)
+{
+	TEEC_Session session = { };
+	uint32_t ret_orig = 0;
+	TEE_OperationHandle op = TEE_HANDLE_NULL;
+	TEE_ObjectHandle keyA = TEE_HANDLE_NULL;
+	TEE_ObjectHandle eph_keyA = TEE_HANDLE_NULL;
+	TEE_ObjectHandle keyB = TEE_HANDLE_NULL;
+	TEE_ObjectHandle eph_keyB = TEE_HANDLE_NULL;
+	TEE_ObjectHandle sv_handle = TEE_HANDLE_NULL;
+	TEE_Attribute params[9] = { };
+	size_t param_count = 0;
+	uint8_t out[128] = { };
+	size_t out_size = 0;
+	uint8_t conf_A[32];
+	uint8_t conf_B[32];
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+		xtest_teec_open_session(&session, &crypt_user_ta_uuid, NULL,
```

Comment:
prefer change indentation at lines 5565, 5582, 5589 5608 and so on.

Reply:
With "proper" indentation (start after opening parenthesis), lines >80 characters cannot be avoided. And many tests use this style already. So I'm not sure which is best.

Reply:
We could rewrite it as:
``` C
res = xtest_teec_open_session(...)
if (!ADBG_EXPECT_TEEC_SUCCESS(c, res))
         return;
```
However, I'm not sure that should be in the scope of this PR.

Reply:
@jenswi-linaro true, we have about 800 occurrences of this pattern... all are more or less formatted like the above.